eslint/babel-eslint-parser/src/client.cjs

```diff
@@ -0,0 +1,67 @@
+const path = require("path");
+
+let send;
+
+exports.getVersion = sendCached("GET_VERSION");
```

Comment:
Could we extract these action types to a constant that we reuse everywhere so we don't risk bugs due to typos?

Reply:
I moved them to this file (rather than always using the string) so that we only have to care about those strings in the two "communication files" (worker client and server). Everywhere else we use `getVersion()` etc.
Also, https://github.com/babel/babel/pull/13199/files#diff-62c6b0f19ec3e004fffbecf4acab02bc005e538ea3c08e9bd558def1f28634c7R32 throws so that we catch any invalid string early.